packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,304 @@
+@ffe-blue-royal-escape: escape('@{ffe-blue-royal}');
+@ffe-blue-royal-fill: @ffe-blue-royal-escape;
+@ffe-blue-azure-escape: escape('@{ffe-blue-azure}');
+@ffe-blue-azure-fill: @ffe-blue-azure-escape;
+
+.ffe-file-upload {
+    margin-top: 45px;
+    display: block;
+
+    @media (min-width: @breakpoint-md) {
+        display: -webkit-box;
```

Comment:
Vendor prefixes are not necessary to include, as they will be generated by autoprefixer.